great_expectations/expectations/expectation.py

```diff
@@ -1365,21 +1369,21 @@ def _validate(
         )
 
 
-class MulticolumnMapExpectation(TableExpectation, ABC):
+class ColumnPairMapExpectation(TableExpectation, ABC):
     map_metric = None
     domain_keys = (
         "batch_id",
         "table",
-        "column_list",
+        "column_A",
+        "column_B",
         "row_condition",
         "condition_parser",
-        "ignore_row_if",
     )
-    success_keys = tuple()
+    success_keys = ("mostly",)
     default_kwarg_values = {
         "row_condition": None,
         "condition_parser": None,  # we expect this to be explicitly set whenever a row_condition is passed
-        "ignore_row_if": "all_value_are_missing",
+        "mostly": 1,
```

Comment:
Great catch. Mostly should either be `None` or a float between 0 and 1. It shouldn't affect the eventual calculation since it will be compared to a float value, but I'd like to vote that this be `1.0` as well